x-pack/plugins/case/server/routes/api/cases/configure/patch_configure.ts

```diff
@@ -67,6 +70,12 @@ export function initPatchCaseConfigure({ caseConfigureService, caseService, rout
           body: CaseConfigureResponseRt.encode({
             ...myCaseConfigure.saved_objects[0].attributes,
             ...patch.attributes,
+            connector: {
+              ...(patch.attributes.connector
+                ? patch.attributes.connector
+                : myCaseConfigure.saved_objects[0].attributes.connector),
```

Comment:
tell if I am wrong but I feel that we need to `transformESConnectorToCaseConnector` here, correct?